nuxt-frontend/components/flutter-app-development/SuccessStorySection.vue

```diff
@@ -0,0 +1,117 @@
+<template>
```

Comment:
It should be sticky with the footer
![Screenshot from 2023-11-09 17-09-50](https://github.com/canopas/canopas-website/assets/123158106/c188eb3a-d5a6-403f-a911-4d79b659f77f)